flask_security/forms.py

```diff
@@ -274,7 +276,8 @@ class ChangePasswordForm(Form, PasswordFormMixin):
 
     new_password_confirm = PasswordField(
         get_form_field_label('retype_password'),
-        validators=[EqualTo('new_password', message='RETYPE_PASSWORD_MISMATCH')])
+        validators=[password_required,
```

Comment:
similar to https://github.com/mattupstate/flask-security/pull/591/files#r100505404